src/main/scripts/data/someicons.json

```diff
@@ -0,0 +1,44 @@
+[
+  "fa-telegram",
+  "fa-envelope-open",
+  "fa-area-chart",
+  "fa-bar-chart",
+  "fa-usd",
+  "fa-shopping-cart",
+  "fa-archive",
+  "fa-cutlery",
+  "fa-beer",
+  "fa-bed",
+  "fa-wifi",
+  "fa-address-book",
+  "fa-id-badge",
+  "fa-child",
+  "fa-user",
+  "fa-comment",
+  "fa-microchip",
+  "fa-database",
+  "fa-desktop",
+  "fa-folder",
+  "fa-film",
+  "fa-industry",
+  "fa-truck",
+  "fa-plane",
+  "fa-globe",
+  "fa-podcast",
+  "fa-info-circle",
+  "fa-thermometer-three-quarters",
+  "fa-adjust",
+  "fa-book",
+  "fa-calculator",
+  "fa-briefcase",
+  "fa-university",
+  "fa-trophy",
+  "fa-bell",
+  "fa-flag",
+  "fa-android",
+  "fa-facebook-square",
+  "fa-twitter",
+  "fa-youtube-play",
+  "fa-music",
+  "fa-picture-o"
+]
```

Comment:
After my commit will be merged (today), you should use constants in future `Const.ts` file